payment_alipay/data/alipay.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data noupdate="0">
+
+        <record id="payment_acquirer_alipay" model="payment.acquirer">
+            <field name="name">Alipay</field>
+            <field name="provider">alipay</field>
+            <field name="company_id" ref="base.main_company"/>
+            <field name="view_template_id" ref="alipay_acquirer_button"/> 
+            <field name="environment">test</field>
+            <field name="pre_msg"><![CDATA[
```

Comment:
Is it demo data?
If this data is specific to Elico, please remove.
Better have parameters in properties/setup